src/run_solver.jl

```diff
@@ -161,27 +185,34 @@ function solve_problems(
         )
       catch e
         @error "caught exception" e
-        push!(
-          stats,
-          [
-            solver_name
-            problem_info
-            :exception
-            Inf
-            Inf
-            0
-            Inf
-            Inf
-            fill(0, ncounters)
-            string(e)
-            fill(missing, length(specific))
-          ],
-        )
+        if !first_problem 
+          push!(
+            stats,
+            [
+              solver_name
+              problem_info
+              :exception
+              Inf
+              Inf
+              0
+              Inf
+              Inf
+              fill(0, ncounters)
+              string(e)
+              fill(missing, length(specific))
+            ],
+          )
+        else
+          push!(fails_since_start, string(e))
+        end
       finally
         finalize(problem)
       end
     end
-    (skipthis && prune) || @info log_row(stats[end, col_idx])
+    ((skipthis && prune) || first_problem) || @info log_row(stats[end, col_idx])
+    if colstats == [:solver_name, :name, :nvar, :ncon, :status, :iter, :elapsed_time, :objective, :dual_feas, :primal_feas,]
+      !first_problem || @info log_row(Any[solver_name, problem.meta.name, problem.meta.nvar, problem.meta.ncon, :exception, 0, Inf, Inf, Inf, Inf])
+    end # We don't print the first problems that have exceptions/skips when colstats is not default... 
```

Comment:
I don't know if my approach is optimal, what might happen here is that colstats does not have its default value, then if the n first iterations are skipped/throw an error, then the first n iterations will not be printed.